doc/programming_guide/pygame_comparison.rst

```diff
@@ -83,10 +83,10 @@ Here are some comparisons between Arcade 3.0 and Pygame 2.2.0 ce:
      - No
    * - Type Hints
      - Yes
-     - No
+     - Yes
    * - Transparency support
      - Yes
-     - Must specify transparent colorkey
+     - Yes
```

Comment:
Does Pygame no longer need to set transparent color key?

Reply:
My apologies, I must've missed the notification for your comment on my end.

Pygame Surfaces can and frequently do have other types of transparency, such as per pixel alpha (use pygame.SRCALPHA or convert_alpha()), and global opacity (set_alpha())

Reply:
Confirmed, loading an image requires no extra steps for transparency support now. Yay!